bigbluebutton-html5/private/config/settings.yml

```diff
@@ -139,6 +139,8 @@ public:
       duration: 4000
     appsGallery:
       maxPinnedApps: 3
+    leaveLabel:
```

Comment:
This would likely make more sense if located in the `layout` [configuration object](https://github.com/bigbluebutton/bigbluebutton/blob/668d133330843cad413ab94ecc4af5a553261246/bigbluebutton-html5/private/config/settings.yml#L816-L822).

E.g.: `public.layout.showLeaveSessionLabel`